gatling/commands.rb

```diff
@@ -0,0 +1,122 @@
+require 'optparse'
+require 'net/http'
+require 'json'
+
+class Babysitter
```

Comment:
![I see what you did there](https://i3.kym-cdn.com/photos/images/original/000/074/461/I-SEE-WHAT-YOU-DID-THERE_HOLYSHIT.jpg)